src/modules/config/config-service.ts

```diff
@@ -0,0 +1,51 @@
+import config from 'config';
+import { Environment, MissingKeyError, ValueTypeMismatchError } from './types';
+
+export enum ConfigType {
```

Comment:
Please move it to internals/config-type.ts

Service should not contain details or should contain very minimal detail so reader of this file may not know much about implementation details please.